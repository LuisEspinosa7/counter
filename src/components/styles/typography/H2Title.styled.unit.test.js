import React from 'react';
import renderer from 'react-test-renderer';
import "jest-styled-components";
import { StyledH2Title } from './H2Title.styled';

test('should render H2Titlte by default - no props', () => {
    const component = renderer.create(
        <StyledH2Title>Example title</StyledH2Title>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    expect(tree).not.toHaveStyleRule("margin");
    expect(tree).not.toHaveStyleRule("font-size");
    expect(tree).not.toHaveStyleRule("font-weight");
    expect(tree).not.toHaveStyleRule("text-shadow");
    expect(tree).not.toHaveStyleRule("color");
})