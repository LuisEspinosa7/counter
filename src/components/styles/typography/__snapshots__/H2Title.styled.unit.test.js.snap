// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render H2Titlte by default - no props 1`] = `
<h2
  className=""
>
  Example title
</h2>
`;
